cli/internal/cloudcmd/tfupgrade_test.go

```diff
@@ -19,58 +19,61 @@ import (
 )
 
 func TestPlanUpgrade(t *testing.T) {
+	defaultFs := func(require *require.Assertions) file.Handler {
+		fs := file.NewHandler(afero.NewMemMapFs())
+		require.NoError(fs.MkdirAll("test"))
```

Comment:
nit: extract "test" as variable templateDir or simply name it so, to help indicate that this is the folder where the TF templates are expected. Then rename `defaultFs` to `fsWithTemplateDir`